leeger/league_loader/YahooLeagueLoader.py

```diff
@@ -169,3 +171,12 @@ def __getOwnerByYahooTeam(self, yahooTeam: YahooTeam) -> Owner:
             return self.__yahooManagerIdToOwnerMap[yahooManagerId]
         raise DoesNotExistException(
             f"Yahoo Manager ID {yahooManagerId} not found in saved Yahoo Manager IDs.")
+
+    # First 3 digits are the game_key, followed by the league_id which is typically 5-6 digits
```

Comment:
NIT: Love the comment for clarity, can we move it inside the method as a doc string?

Just in between """ """